pygmt/src/contour.py

```diff
@@ -112,6 +118,12 @@ def contour(self, x=None, y=None, z=None, data=None, **kwargs):
     {i}
     {p}
     {t}
+    {b}
+    {d}
+    {e}
+    {f}
+    {h}
+    {U}
```

Comment:
Could you please sort these in ASCIIbetical order? I.e. ABC...XYZabc...xyz. Thanks!

Reply:
Done, thanks @weiji14 !